src/Copy/Text.elm

```diff
@@ -227,7 +227,7 @@ t key =
             "I refuse to be told that nothing can be done and I want change for other victims. I know first-hand how devastating the effects of economic abuse can be."
 
         ExpandQuoteButton ->
-            "but things got better"
+            "keep reading"
```

Comment:
I think this is also part of another card, actually... https://trello.com/c/MrGjEWk2/446-as-a-woman-experiencing-difficult-times-i-need-the-copy-on-the-front-page-to-reflect-the-reality-of-my-situation-so-that-i-dont

Maybe reset this file to `release` to avoid issues later?

Reply:
I've kept this because I want to lock down the layout soon..